config/vmdb.tmpl.yml

```diff
@@ -286,6 +286,11 @@ workers:
       :thread_shutdown_timeout: 10.seconds
     :event_catcher:
       :defaults:
+        :flooding_monitor_enabled: false
+        :flooding_monitor_window: 5.minutes
```

Comment:
These parameters are kind of randomly chosen. We can discuss to tune them more meaningful.